src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -270,12 +270,15 @@ namespace ts.server.typingsInstaller {
             this.installRunCount++;
 
             // send progress event
-            this.sendResponse(<BeginInstallTypes>{
+            const response: BeginInstallTypes = {
                 kind: EventBeginInstallTypes,
                 eventId: requestId,
                 typingsInstallerVersion: ts.version, // qualified explicitly to prevent occasional shadowing
-                projectName: req.projectName
-            });
+                projectName: req.projectName,
+                // TODO: GH#18217 (property was not present)

```

Comment:
why is this a problem?

Reply:
I was just wondering if maybe it *should* be present.